plasma_framework/docs/design/exit_bond_and_bounty_mechanics.md

```diff
@@ -0,0 +1,196 @@
+# Exit Bond and Bounty Mechanics
+
+This document is a description of the current implementation of exit bonds and process exit bounties which are an integral part of the exit mechanism.
+### Terminology
+
+- **Exit Initiator** The person that starts an exit by either calling `startStandardExit()` or piggybacking on an in-flight exit
```

Comment:
Nit pick, it's probably better to use `user` than `person` as the term is more generic.